.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+Thanks for contributing to RIOT !
```

Comment:
If I remember correctly, there shouldn't be a space between the last word and interpunction at the end of a sentence, so: "Thanks  for contributing to RIOT!" (same for other lines, e.g. with question marks).